### ISSUE TITLE
"No files matching the pattern" when using Windows

### ISSUE THREAD

**Issue by nzacca** [stylelint/stylelint#5122]
> Clearly describe the bug

When using stylelint on Windows using a file path with a normal Windows path ex. `C:\dev\myfile.scss`, stylelint reports "No files matching the pattern". This is the same issue that was reported in https://github.com/stylelint/stylelint/issues/4772 but that issue was closed as not enough information was provided.

This _does_ work when forward slashes are used but this is not the standard file path used by Windows. This also affects other tools such as `@speedy/build-angular` which uses stylelint under the hood to pass a list of files to lint.

> Which rule, if any, is the bug related to?

This is not related to any specific Rule. This is about running stylelint successfully on a Windows platform.

> What code is needed to reproduce the bug?

N/A but a simple file may be used such as:

```scss
.foo { color: black; }
```

> What stylelint configuration is needed to reproduce the bug?

N/A but the following can be used to match the example above.

```json
{
  "rules": {
    "color-named": "never"
  }
}
```

> Which version of stylelint are you using?

`13.9.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

Using the CLI with the following command: `stylelint c:\dev\myfile.scss`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

I am using SCSS but this is not a bug with the parsing of the file.

> What did you expect to happen?

I expect the command to complete successfully when using standard Windows file path with backslashes.

> What actually happened (e.g. what warnings or errors did you get)?

The following error message was written to the console:

```shell
Error: No files matching the pattern "c:\dev\myfile.scss" were found.
    at c:\dev\node_modules\stylelint\lib\standalone.js:212:12
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

Notes:

To fix this, I think stylelint just needs to normalize the file paths for the files passed into it to a format that it expects.

**Comment by hudochenkov** [comment]
Could you try to put a file path in double quotes?

```
stylelint "c:\dev\myfile.scss"

```

**Comment by nzacca** [comment]
@hudochenkov 

Thank you so much for the quick reply. I have tried placing the file in quotes and I receive the same error as in the original post.

Since posting, I have tried 13.8.0 and receive the same issue. I'm going to try 13.7.2 next.

**Comment by nzacca** [comment]
@hudochenkov 

I have also tried 13.7.2 now and same problem. I'm wondering now if this is possibly a dependency issue that could be causing this?

**Comment by amiramix** [comment]
I started seeing this exact error after upgrading stylelint from 10.0.1 to 13.13.0. So a temporary workaround is to revert to an older version until this is fixed.

**Comment by glat** [comment]
It works by using slashes "/" in paths. If I use backslashes "\" I got that errors. Windows, of course. Can it be a bug of a base package?

**Comment by woutervanvliet** [comment]
We're seeing this issue as well on our team.

**Comment by woutervanvliet** [comment]
Apparently there's a change in the globby API causing this issue. My colleague is preparing a PR.

**Comment by tbowmo** [comment]
As mentioned by @woutervanvliet  the problem lies in globby, from version 10 it doesn't support backward slashes in globs, which is stated in their documentation [here](https://github.com/sindresorhus/globby#api)

A quick hack where backslashes where replaced with forward slashes on the fileList supplied to globby in standalone.js, makes a couple of tests break (when running the tests on both windows and linux)

**Comment by jeddy3** [comment]
Fixed by https://github.com/stylelint/stylelint/pull/5386

**Comment by zcqno1** [comment]
It seems that the fix was not released yet. Should we wait unti stylint v14 ? @jeddy3 

> Fixed by #5386

**Comment by jeddy3** [comment]
Yes, but we need help getting [14.0.0 ready for release](https://github.com/stylelint/stylelint/issues/5205).

The next task is adding a `overrides` configuration object property. Please consider [contributing](https://stylelint.io/contributing) if you have time.

**Comment by robertYang1024** [comment]
I updated  lint-staged to  "10.0.0" and  solved this problem
